README.md

```diff
@@ -40,8 +40,8 @@ hard-coded defaults and the module parameter values specified on the command lin
 | EVENT_LOG | USER_BLOCK              | {any}                       | If enabled, log user blocked events to the event log (default: enabled=false)       |
 | {any}     | {any}                   | login.fail.attempts         | Number of login attempts before block user account (default: value=5)               |
```

Comment:
The table row should consist out of:
| {any} | login.fail.attempts | {any}  |  Number of login attempts ...
compare in mockHttpCalls()
------
mockServer.stubFor(
      WireMock.get(WireMock.urlPathEqualTo("/configurations/entries"))
        .withQueryParam("query", WireMock.equalTo("**code==login.fail.attempts**"))
        .willReturn(WireMock.okJson(new JsonObject()
          .put("totalRecords", 1)
          .put("configs", new JsonArray().add(new JsonObject().put("value", 5)))
          .encode()))
    );
------
...